test/e2e/daemon_restart.go

```diff
@@ -255,7 +255,7 @@ var _ = Describe("DaemonRestart", func() {
 	})
 
 	It("Controller Manager should not create/delete replicas across restart", func() {
```

Comment:
this test has been skipped by pure luck on gke-ci because it's listed under "flaky" tests.

the reason the other tests started running is because we mistakenly started running tests in the release-1.1 branch instead of master.